### PR TITLE
📝 add section videos

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

:pencil: add section videos

 

### Details

- I created a section where we can share videos and materials when talking or doing tutorials at conferences about pyvista.
- I am referring to the mayavi's [this page](https://mayavi.readthedocs.io/en/latest/misc.html#tutorial-videos).

[ReadTheDocs](https://pyvista-doc--40.org.readthedocs.build/en/40/#videos) :point_left: check RTD html build on this branch

### CODE REVIEW

**Comment by tkoyama010** [comment]
https://pyvista-doc--40.org.readthedocs.build/en/40/#videos :point_left: check build html on this branch

**Comment by tkoyama010** [comment]
It is ready for review :rocket:

File: docs/index.rst [hunk 1]
```diff
@@ -156,6 +156,17 @@ If you are using PyVista in your scientific research, please help our scientific
 visibility by citing our work! Head over to :ref:`citation_ref` to learn more
 about citing PyVista.
 
+Videos
+======
+
+Here are some videos that you can watch to learn pyvista:
+
+- SPyConJP2020 talk "How to plot unstructured mesh file on Jupyter Notebook" (15 minutes):
```
Comment (GuillaumeFavelier) [inline docs/index.rst:164]: Is it `SPyConJP2020` or `PyConJP2020` ? (without the 'S')

**Review by GuillaumeFavelier — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (tkoyama010) [reply docs/index.rst:164]: Oops. I had typo. Thanks :+1:
```suggestion
- PyConJP2020 talk "How to plot unstructured mesh file on Jupyter Notebook" (15 minutes):
```

**Review by GuillaumeFavelier — APPROVED** [review]
LGTM, what about the other @pyvista/developers ?

**Comment by akaszynski** [comment]
Watched the videos and they looked great!  Thanks for the addition.

**Review by akaszynski — APPROVED** [review]

**Pull request merged**